### PR TITLE
travis: test on ubuntu focal

### PR DESCRIPTION
**Pull request by thijstriemstra**

### CODE REVIEW

**Review by rm-hull — DISMISSED** [review]

**Comment by rm-hull** [comment]
huh weird ... 3 hours later and no builds have gone through: Is travis having issues or is it because the PR is marked as draft ? 

in any regard, i've noticed that builds are taking a long time to start on travis these days...

**Comment by thijstriemstra** [comment]
the builds did start: https://travis-ci.org/github/rm-hull/luma.core/jobs/738983807

but some weird virtualenv error..

**Comment by rm-hull** [comment]
See also https://travis-ci.community/t/build-delays-for-open-source-project/10272/9

**Comment by thijstriemstra** [comment]
We're currently modifying the travis system python packages by upgrading pip, setuptools, tox and travis libraries.

It's probably better to create a separate virtualenv and upgrade everything there. Tox will then create separate virtualenvs during it's testing. This way we don't have to fiddle with the system and potentially break something.

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ matrix:
         - python: 3.8
           env: TOXENV=qa,doc
 
-install: pip install --ignore-installed --upgrade setuptools pip tox coveralls
+install:
+  - pip uninstall virtualenv
```
Comment (rm-hull) [inline .travis.yml:19]: see: https://travis-ci.org/github/rm-hull/luma.core/jobs/740002770#L202-L207

Add a `-y` flag maybe ?

Comment (thijstriemstra) [reply .travis.yml]: ill rework this pr this weekend. that last commit was a (dumb) test.

**Review by rm-hull — APPROVED** [review]

**Pull request merged**